docs/current-cli-contract.md

```
@@ -1123,7 +1200,7 @@ enabled.
   VSPreview-confirmed results still write to the shared reuse cache when
   `cache_results = true`. `prompt` shows a Rich stderr table for a complete
   valid VSPreview-confirmed offset set and asks
-  `Reuse previous preview-confirmed alignment offsets? [y/N]`; default, EOF,
+  `Reuse these offsets? [y/N]: `; default, EOF,
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)